static/src/javascripts/projects/common/utils/has-parent.js

```diff
@@ -0,0 +1,21 @@
+define([
+    'qwery'
+], function (
+    qwery
+) {
+
+    function hasParent(selector, context) {
+        while (!context[0].classList.contains(selector)) {
```

Comment:
if the first check is for the selector to apply to the context, i think you'll get a false positive for elements for which that is `true`, but for who's parents it's `false`